src/main/resources/application.properties

```diff
@@ -37,9 +37,9 @@ endpoint.mongodb.disqualifications_collection=mongodb:mongo?database=${ENDPOINT_
 endpoint.log.output=log:output
 results.initial.capacity=${RESULTS_INITIAL_CAPACITY}
 
-endpoint.s3.upload=aws2-s3://${RESULTS_BUCKET}
-endpoint.s3presigner.download=aws2-s3://${RESULTS_BUCKET}?operation=createDownloadLink
-aws.expiry=2000
+endpoint.s3.upload=aws2-s3://${RESULTS_BUCKET}?accessKey=${AWS_ACCESS_KEY_ID}&secretKey=${AWS_SECRET_ACCESS_KEY}&region=${AWS_REGION}
```

Comment:
Just make sure that any calling class of this URL suppresses the above url from the end user as there is a chance (Depending on implementation) that the Access_Key and Secret could bubble up through an inner stack as a web request.

Reply:
During startup, if the endpoint is misconfigured, then the endpoint with unresolved values will be output to the logs (e.g. `Could not resolve placeholder 'AWS_REGION' in value "aws2-s3://${RESULTS_BUCKET}?accessKey=${AWS_ACCESS_KEY_ID}&secretKey=${AWS_SECRET_ACCESS_KEY}&region=${AWS_REGION}`). Any exceptions thrown by the Camel S3 component at runtime should not expose sensitive information.

Reply:
Ok no problem then it all looks good :-)